recipes/pybind_json/bld.bat

```diff
@@ -0,0 +1,12 @@
+cmake -G "NMake Makefiles" ^
```

Comment:
It seems that there is no package ``nmake`` for ``windows``. You can try to build it using ninja

Reply:
```suggestion
cmake -G "Ninja" ^
```